test/integration/AuthenticatedFileResourceStore.test.ts

```diff
@@ -16,6 +17,8 @@ describe('A server using a AuthenticatedFileResourceStore', (): void => {
   let rootFilePath: string;
 
   beforeAll(async(): Promise<void> => {
+    setGlobalLoggerFactory(new VoidLoggerFactory());
```

Comment:
For my info, what happens if we don't?

@joachimvh Can't we set this somewhere globally for all tests? Test initializer? I really think we need to; it's unmanageable otherwise.

Reply:
Also going to tag @rubensworks since he might know more about this.

Would have to check but pretty sure there are going to be errors because no logger was configured.

We can check if there is some way to have jest execute certain code before any tests get started. Or we could make the logger so that it is a VoidLogger by default if no logger was configured.

Reply:
If no logger is configured, the tests fail due to
```
throw new Error('No logger factory has been set. Can be caused by logger invocation during initialization.');
```

Reply:
Yeah, we have to configure a logger. Just globally, not in every test.

Reply:
Do you mean something like this https://github.com/solid/community-server/pull/272/commits/a69cfcc293d4fcbae2ab914cba26f55e37c26962 ?
When I read the [documentation](https://jestjs.io/docs/en/configuration#setupfilesafterenv-array) I believe that this setup file is run for each test file. Is this a problem? If so, I'll see if this can only be done once.

Reply:
Yeah, that's fine.